scripts/build.sh

```diff
@@ -0,0 +1,26 @@
+IDIR =../include
```

Comment:
Codacy found an issue: [Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6296785)